src/main/kotlin/uk/gov/gdx/datashare/controller/EventsController.kt

```diff
@@ -222,6 +275,26 @@ class EventsController(
       throw e
     }
   }
+
+  private fun eventLink(id: UUID): WebMvcLinkBuilder =
+    linkTo(methodOn(EventsController::class.java).getEvent(id))
+
+  private fun eventsLink(
+    eventTypes: List<String>? = null,
+    startTime: LocalDateTime? = null,
+    endTime: LocalDateTime? = null,
+    pageNumber: Int = 0,
```

Comment:
Presumably the Spring API here doesn't support `Pageagle` outside of the JPA layer? I'd be tempted to use its/something structured as much as possible